src/app/legacy/components/RadioSchedule/ScheduleItemHeader/index.jsx

```diff
@@ -85,40 +73,35 @@ const ScheduleItemHeader = ({
     ['media', 'listenNext'],
     translations,
   );
-
   const listenLabelTranslations = {
     live: listenLive,
     next: listenNext,
     onDemand: listen,
   };
-
   const formattedStartTime = formatUnixTimestamp({
     timestamp: startTime,
     format: 'HH:mm',
     timezone,
     locale,
     isRelative: false,
   });
-
   const formattedDuration = detokenise(
     durationLabel,
     durationDictionary({ duration, locale }),
   );
-
   const episodeTitle = formatUnixTimestamp({
     timestamp: startTime,
     format: 'LL',
     timezone,
     locale,
     isRelative: false,
   });
-
   const content = (
     // This is a temporary fix for the a11y nested span's bug experienced in TalkBack, refer to the following issue: https://github.com/bbc/simorgh/issues/9652
     // eslint-disable-next-line jsx-a11y/aria-role
     <span role="text" id={`scheduleItem-${id}`}>
       <VisuallyHiddenText>{`${listenLabelTranslations[state]}, `}</VisuallyHiddenText>
-      {isLive && <LiveLabel liveText={liveLabel} ariaHidden />}
+      {isLive && <LiveLabel id={`radio-schedule-${id}`} />}
```

Comment:
I think we should keep the behaviour as it was before, and remove the id.
```suggestion
      {isLive && <LiveLabel />}
```

Reply:
We put it back to how it was before.